web/main/settings.py

```diff
@@ -42,7 +43,9 @@
 
     'typogrify',
     'social_auth',
+    'adminsortable2',
     'django_gravatar',
+    'django_extensions',
```

Comment:
Why do you need `django-extensions` ? I remembered that we tried to use this package. but we did not use it. maybe we forgot to remove it in requirements file.

Reply:
Maybe we can use in the local environment.

Reply:
It has some helpful features that can be used in development like `python manage.py show_urls`. Doesn't really relate to this issue, but would be helpful to have it in the future.

Reply:
I agree with you. we can use it .